src/improvements/tasks/sep/000-opcm-upgrade-13/README.md

```diff
@@ -0,0 +1,36 @@
+# 000-opcm-upgrade-13: 
```

Comment:
Lets split this over 2 PRs, one for the template (first), then one for the task.

Also in the 2nd PR, you can remove these notes in the README.md file. This will have to be properly filled out. The values in the config.toml file also need to be decided upon.